packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2431,6 +2437,15 @@ class InputDecoration {
   /// The style to use for the [helperText].
   final TextStyle helperStyle;
 
+  /// The maximum number of lines the [helperText] can occupy.
+  ///
+  /// Defaults to null, which means that the [helperText] will be limited
+  /// to a single line with [TextOverflow.ellipsis].
+  ///
+  /// This value is passed along to the [Text.maxLines] attribute
+  /// of the [Text] widget used to display the error.
```

Comment:
"used to display the **helper**"

Reply:
What do you think about having `helperMaxLines` and `errorMaxLines` link to each other?

```dart
///
/// See also: [errorMaxLines]
```

And the same thing for errorMaxLines but linking to [helperMaxLines].  Might be helpful because if you're setting one you might want to set both.